composer.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "keboola/processor-filter-files",
+    "license": "MIT",
+    "description": "Filters only files matching a given mask. Files are filtered recursively.",
+    "require": {
+        "PHP": "^7.1",
+        "keboola/php-component": "^3.0"
+    },
+    "require-dev": {
+        "jakub-onderka/php-parallel-lint": "^1.0",
+        "keboola/coding-standard": "^2.0",
+        "keboola/php-temp": "^1.0",
+        "phpstan/phpstan-shim": "^0.9.2",
+        "phpunit/phpunit": "^7.0",
+        "symfony/process": "^4.0",
+        "keboola/datadir-tests": "^1.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "MyComponent\\": "src/"
```

Comment:
Namespace

Reply:
Tím jsem si nebyl jistej - vlastně mi přišlo dobrý to nechat tak, jak mi to připraví generátor?

Reply:
Hotovo

Reply:
a pritom jsi sam byl proti https://keboola.slack.com/archives/C9FEAHW7N/p1521646041000160 :D

Reply:
jj! chtěl jsem to vyzkoušet a bejt trochu línej :-)